.github/workflows/gajira.yml

```diff
@@ -0,0 +1,43 @@
+name: Raise Jira Issue
+
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  gajira:
+    name: Raise Jira Issue
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check if pull request was created by Snyk
+      if: ${{ !contains(github.event.pull_request.labels, 'snyk') }}
```

Comment:
Would it be possible to move this check to the job level, rather than performing it as a step?

This will stop the job from running if it shouldn't, rather than failing.

This method is used here: [https://github.com/ministryofjustice/laa-access-civil-legal-aid/blob/main/.github/workflows/gajira.yml#L11](https://github.com/ministryofjustice/laa-access-civil-legal-aid/blob/main/.github/workflows/gajira.yml#L11)